narwhals/_pandas_like/series.py

```diff
@@ -112,7 +113,8 @@ def __getitem__(self, idx: int) -> Any: ...
     def __getitem__(self, idx: slice | Sequence[int]) -> Self: ...
 
     def __getitem__(self, idx: int | slice | Sequence[int]) -> Any | Self:
-        if isinstance(idx, int):
+        np = get_numpy()
+        if np.isscalar(idx):
```

Comment:
Should this be a one liner to also check numpy is installed?

```suggestion
        if isisntance(idx, int) or ((np := get_numpy()) is not None and np.isscalar(idx)):
```

Reply:
This is pandas-specific code, so isn't numpy guaranteed to be installed? Still, there's no harm checking.
You have a typo: could you please change `isisntance` to `isinstance`? Then I will commit your suggestion.
If you know of a more general way to check for scalars, that might be preferable.

Reply:
Yeah sorry, I realized that after the comment, apology for the inconvenience! I am resolving the comment